ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -429,24 +429,71 @@ func (r *RayJobReconciler) getOrCreateRayClusterInstance(ctx context.Context, ra
 	rayClusterInstance := &rayv1alpha1.RayCluster{}
 	err := r.Get(ctx, rayClusterNamespacedName, rayClusterInstance)
 	if err == nil {
-		r.Log.Info("RayJob associated rayCluster found", "rayjob", rayJobInstance.Name, "raycluster", rayClusterNamespacedName)
-		// just return the rayClusterInstance if cluster spec is nil
-		if rayJobInstance.Spec.RayClusterSpec == nil {
+		r.Log.Info("Found associated RayCluster for RayJob", "rayjob", rayJobInstance.Name, "raycluster", rayClusterNamespacedName)
+
+		// Case1: The job is submitted to an existing ray cluster, simply return the rayClusterInstance.
```

Comment:
```		
// Case1: The job is submitted to an existing RayCluster. Return the existing RayCluster instance and ignore all updates from RayClusterSpec.
```

Reply:
This is becasue in golang, we can't take the address of a constant(Constants don't have a specific memory location). We need to define a variable, assign the constant value to the variable, and then take the address of that variable.